src/query/interpret/awesome_memgraph_functions.cpp

```diff
@@ -638,6 +649,9 @@ TypedValue ValueType(const TypedValue *args, int64_t nargs, const FunctionContex
       return TypedValue("DURATION", ctx.memory);
     case TypedValue::Type::Enum:
       return TypedValue("ENUM", ctx.memory);
+    case TypedValue::Type::Point2d:
+    case TypedValue::Type::Point3d:
+      return TypedValue("POINT", ctx.memory);
```

Comment:
Since opencypher9 doesn't define point, did you maybe check what neo4j does here?

Reply:
It says `POINT NOT NULL`